layers/+email/mu4e/README.org

```diff
@@ -116,6 +117,42 @@ the given account variables.
            "All inboxes" ?i)))
 #+END_SRC
 
+** Notifications
+~mu4e-alert~ is an extension that provides desktop notifications and adds the
+count of unread messages to the modeline.
+
+[[https://raw.githubusercontent.com/iqbalansari/mu4e-alert/master/screenshots/mu4e-alert-in-action.png]]
+
+To enable notifications about new messages, add the following line to your
+~dotspacemacs/user-config~:
+
+#+BEGIN_SRC emacs-lisp
+  (setq mu4e-enable-notifications t)
+#+END_SRC
+
+By default, notifications will be shown in the ~*Messages*~ buffer. To enable
+desktop notifications about new messages, add the following lines to
+your ~dotspacemacs/user-config~, according to your operating system and the
+installed libraries:
+
+#+BEGIN_SRC emacs-lisp
+  (with-eval-after-load 'mu4e-alert
+    ;; Enable Desktop notifications
+    (mu4e-alert-set-default-style 'notifications)) ; For linux
+    ;; (mu4e-alert-set-default-style 'libnotify))  ; Alternative for linux
+    ;; (mu4e-alert-set-default-style 'growl))      ; For Mac OSX
+    ;; (mu4e-alert-set-default-style 'notifier))   ; Alternative for Mac OSX
```

Comment:
It'd be nice to point out that `terminal-notifier` is the needed app to get `'notifier` working on OS X.

Perhaps `growl` should be presented as the alternative as `notifier` is more popular now I guess, and it is what the ERC layer use for OS X.